devsetup/tripleo/config-download-multistack.yaml

```diff
@@ -0,0 +1,149 @@
+        tags:
+          - 192.168.122.0/24
+    compute-0-ctlplane:
+      fixed_ips:
+        - ip_address: 192.168.122.106
+      subnets:
+        - cidr: 192.168.122.0/24
+      network:
+        tags:
+          - 192.168.122.0/24
+  NodePortMap:
+    controller-0: # overcloud-controller-0 (cell0)
+      ctlplane:
+        ip_address: 192.168.122.103
+        ip_address_uri: 192.168.122.103
+        ip_subnet: 192.168.122.0/24
+      internal_api:
+        ip_address: 172.17.0.103
+        ip_address_uri: 172.17.0.103
+        ip_subnet: 172.17.0.0/24
+      storage:
+        ip_address: 172.18.0.103
+        ip_address_uri: 172.18.0.103
+        ip_subnet: 172.18.0.0/24
+      storage_mgmt:
+        ip_address: 172.20.0.103
+        ip_address_uri: 172.20.0.103
+        ip_subnet: 172.20.0.0/24
+      tenant:
+        ip_address: 172.19.0.103
+        ip_address_uri: 172.19.0.103
+        ip_subnet: 172.19.0.0/24
+    compute-1: # cell1-compute-0
+      ctlplane:
+        ip_address: 192.168.122.104
+        ip_address_uri: 192.168.122.104
+        ip_subnet: 192.168.122.0/24
+      internal_api:
+        ip_address: 172.17.1.104
+        ip_address_uri: 172.17.1.104
+        ip_subnet: 172.17.1.0/24
+      storage:
+        ip_address: 172.18.1.104
+        ip_address_uri: 172.18.1.104
+        ip_subnet: 172.18.1.0/24
+      tenant:
+        ip_address: 172.19.1.104
+        ip_address_uri: 172.19.1.104
+        ip_subnet: 172.19.1.0/24
+    compute-2: # cell2-controller-compute-0 AIO
+      ctlplane:
+        ip_address: 192.168.122.105
+        ip_address_uri: 192.168.122.105
+        ip_subnet: 192.168.122.0/24
+      internal_api:
+        ip_address: 172.17.2.105
+        ip_address_uri: 172.17.2.105
+        ip_subnet: 172.17.2.0/24
+      storage:
+        ip_address: 172.18.2.105
+        ip_address_uri: 172.18.2.105
+        ip_subnet: 172.18.2.0/24
+      storage_mgmt:
+        ip_address: 172.20.2.103
+        ip_address_uri: 172.20.2.103
+        ip_subnet: 172.20.2.0/24
+      tenant:
+        ip_address: 172.19.2.105
+        ip_address_uri: 172.19.2.105
+        ip_subnet: 172.19.2.0/24
+    compute-0: # cell1-controller-0
+      ctlplane:
+        ip_address: 192.168.122.106
+        ip_address_uri: 192.168.122.106
+        ip_subnet: 192.168.122.0/24
+      internal_api:
+        ip_address: 172.17.1.106
+        ip_address_uri: 172.17.1.106
+        ip_subnet: 172.17.1.0/24
+      storage:
+        ip_address: 172.18.1.106
+        ip_address_uri: 172.18.1.106
+        ip_subnet: 172.18.1.0/24
+      tenant:
+        ip_address: 172.19.1.106
+        ip_address_uri: 172.19.1.106
+        ip_subnet: 172.19.1.0/24
+  CtlplaneNetworkAttributes:
+    network:
+      dns_domain: localdomain
+      mtu: 1500
+      name: ctlplane
+      tags:
+        - 192.168.122.0/24
+    subnets:
+      ctlplane-subnet:
+        cidr: 192.168.122.0/24
+        dns_nameservers: 192.168.122.1
+        gateway_ip: 192.168.122.1
```

Comment:
those need to be configurable, as in CI we use ``standalone_gateway: 192.168.122.10``